lib/emque/consuming/version.rb

```diff
@@ -1,5 +1,5 @@
 module Emque
   module Consuming
-    VERSION = "1.0.0.beta4"
+    VERSION = "1.0.0.beta5"
```

Comment:
to follow-up, this will be the 1.1.0 release after we get an official 1.0.0 up with the current code